Build.mak

```diff
@@ -24,7 +24,9 @@ $O/allunittests $O/fastunittest: $O/libvibed.a $O/libdyaml.a
 $O/%unittests: override LDFLAGS += -L$O -lvibed -ldyaml -levent -lssl -lcrypto
 $O/pkg-neptune.stamp: release overview
 
-%unittests: $O/libvibed.a $O/libdyaml.a
```

Comment:
It is quite confusing in this change that some unrelated dependencies are removed, specially when in the diff context you can see they are already there, so I would separate this small change in a different commit just for clarity.